### DISCUSSION TITLE
Help with installation

### DISCUSSION THREAD

**Discussion by rwegener2** [pythonspeed/filprofiler#353] [Q&A] [answered]
I've read through the docs and I am so excited about this package!

To get started I installed with `conda install -c conda-forge filprofiler` in my conda env, restarted my shell (just in case), then ran `fil-profile run example.py`.  For `example.py` I have the example [from the documentation](https://pythonspeed.com/fil/docs/trying.html).

When doing this I get this error:
```
dyld[6163]: re-export ordinal -2 in /Users/rwegener/.pyenv/versions/miniconda3-4.7.12/envs/pangeo-forge-recipes/lib/python3.9/site-packages/filprofiler/_filpreload.cpython-39-darwin.so out of range for _aligned_allocsymbol not found in flat namespace '_aligned_alloc'
[1]    6163 abort      fil-profile run example.py
```

Does anyone see what I am doing wrong? Thanks so much!

**Comment by itamarst** [comment, marked as answer]
Looks like I did something wrong, somewhere... sorry it didn't work, and thanks for reporting this.

I will open a bug, but as a short term workaround, try uninstalling the conda package and doing `pip install filprofiler` instead, and see if that works.

**Reply by rwegener2** [reply]
Okay, thanks so much! `pip` works great.
